pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -640,9 +640,10 @@ func (k *Kubernetes) Undeploy(komposeObject kobject.KomposeObject, opt kobject.C
 			//FIXME: gracePeriod is nil
 			err = rpDeployment.Stop(namespace, t.Name, TIMEOUT*time.Second, nil)
 			if err != nil {
-				return err
```

Comment:
Whole point of the issue is that we error out (return error) and that is not good. We need at least try delete the rest of the objects. Error in deleting one of the objects shouldn't be show stopper for deleting the rest.

On the other hand, `Undeploy` function should probably return error if we can't be sure that all the objects were deleted. Maybe those error should be aggregated so it can be returned at the end.

@containscafeine was looking into error handling lately. He might know what would be best here.